collector/README.md

```diff
@@ -61,93 +61,96 @@ marked with a '?' in the `compare` page.
-  The possible choices are one or more (comma-separated) of `Check`, `Debug`,
-  `Opt`, and `All` (the default).
-- `--runs $RUNS` can be used to select what profiling runs are done for each
-  build. The possible choices are one or more (comma-separated) of `Full`,
-  `IncrFull`, `IncrUnchanged`, `IncrPatched`, and `All` (the default). Note
-  that `IncrFull` is always run (even if not requested) if either of
-  `IncrUnchanged` or `IncrPatched` are run.
-
-### Comparing different versions on your own machine
+The following options alter the behaviour of the `bench_local` subcommand.
+- `--builds <BUILDS>`: the build kinds to be benchmarked. The possible choices
+  are one or more (comma-separated) of `Check`, `Debug`, `Doc`, `Opt`, and
+  `All`. The default is `Check,Debug,Opt`.
+- `--cargo <CARGO>`: a path (relative or absolute) to a Cargo executable that
+  will be used to build the benchmarks. By default, the nightly Cargo installed
+  by `rustup` will be used. This is usually fine, though in rare cases it may
+  cause local results to not exactly match production results, because Cargo
+  sometimes begins passing (or stops passing) various flags to rustc.
+- `--db $DATABASE`: a path (relative or absolute) to a sqlite database file in
+  which the timing data will be placed. It will be created if it does not
+  already exist. The default is `results.db`. Alternatively, the collector
+  supports postgres as a backend and the URL can be specified (beginning with
+  `postgres://`), but this is unlikely to be useful for local collection.
+- `--exclude <EXCLUDE>`: this is used to run a subset of the benchmarks. The
+  argument is a comma-separated list of strings. When this option is specified,
+  a benchmark is excluded from the run if its name contains one or more of the
+  given strings.
+- `--include <INCLUDE>`: the inverse of `--exclude`. The argument is a
+  comma-separated list of strings. When this option is specified, a benchmark
+  is included in the run only if its name contains one or more of the given
+  strings.
+- `--runs $RUNS`: the run kinds to be benchmarked. The possible choices are one
+  or more (comma-separated) of `Full`, `IncrFull`, `IncrUnchanged`,
+  `IncrPatched`, and `All`. The default is `All`. Note that `IncrFull` is
+  always run (even if not requested) if either of `IncrUnchanged` or
+  `IncrPatched` are run.
+- `--rustdoc <RUSTDOC>`: a path (relative or absolute) to a rustdoc
+  executable that will be benchmarked (but only if a `Doc` build is requested
+  with `--builds`). If a `Doc` build is requested, by default the tool will
+  look for a rustdoc executable next to the rustc specified via the `<RUSTC>`
+  argument.
+- `--self-profile`: enable self-profiling, i.e. the inclusion of query
+  statistics in the output. The `measureme` tool must be installed for this to
+  work.
+
+`RUST_LOG=debug` can be specified to enable verbose logging, which is useful
```

Comment:
nit: you probably at least want `RUST_LOG=collector=debug`, otherwise you'll get e.g. tokio debugging which is not useful.

Reply:
Doesn't seem to make a difference.